### DISCUSSION TITLE
It's almost absurd how useful this project is. Thanks.

### DISCUSSION THREAD

**Discussion by dpowerstsi** [WinMerge/winmerge#2213] [General]
I just donated $25 from the website link because of how useful WinMerge is. It's just absurdly useful and well-written. Thanks to the whole dev team. You're making terrific OSS.

**Comment by sdottaka** [comment]
Thank you for your donation and positive feedback!

**Comment by meteorquake** [comment]
Totally agree, I've practically worn the program out through over-use, if that were possible!
d
